assets/sass/components/_content-tile-small.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-hub-contentTileSmall {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 30;
```

Comment:
can we use a gov uk value for consistency?

Reply:
This class `govuk-!-margin-6` should cover it

Reply:
turns out the margin style isn't actually needed